src/components/ui/font-family-toolbar-button.tsx

```
@@ -0,0 +1,88 @@
+  { value: 'Microsoft Sans Serif', label: 'Microsoft Sans Serif' },
+  { value: 'Palatino Linotype', label: 'Palatino Linotype' },
+  { value: 'Roboto', label: 'Roboto' },
+  { value: 'Segoe UI', label: 'Segoe UI' },
+  { value: 'Tahoma', label: 'Tahoma' },
+  { value: 'Times New Roman', label: 'Times New Roman' },
+  { value: 'Trebuchet MS', label: 'Trebuchet MS' },
+  { value: 'Verdana', label: 'Verdana' },
+];
+
+export function FontFamilyToolbarButton() {
+  const { editor, api } = useEditorPlugin(FontFamilyPlugin);
```

ESLint

[error] 'editor' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)